File: Python/Arrays.py
def AllArrays():
    one = "For all the following functions go to"
    two = "\nhttps://github.com/AustinGitHub/Programming-Functions/wiki/Array-Elements"
    three = "\nRight click and open url"
    return one+two+three
def AddAll(a):
    total = 0
    for x in a:
        total += x
    return total
def Highest(a):
    highest = 0;
    for x in a:
        if(x > highest):
            highest = x
    return highest;
def Lowest(a):
    lowest = Highest(a);
    for x in a:
        if(x < lowest):
            lowest = x
    return lowest;

File: JavaScript/Arrays.js
function AllArrays(){
	return "For all the following functions go to https://github.com/AustinGitHub/Programming-Functions/wiki/Array-Elements";
}
function AddAll(a){
	var total = 0;
	for(var x=0;x<a.length;x++){
		total += a[x];
	}
	
	return total;
}
function Highest(a){
	var Highest = 0;
	for(var x=0;x<a.length;x++){
		if(a[x] > Highest){
			Highest = a[x];
		}
	}
	return Highest;
}
function Lowest(a){
	var Lowest = Highest(a);
	for(var x=0;x<a.length;x++){
		if(a[x] < Lowest){
			Lowest = a[x];
		}
	}
	return Lowest;

}